### PR TITLE
add sorting functionality

### PR DESCRIPTION
**Pull request by captainread**
drop down for sorting type, radio buttons for asc/desc

### CODE REVIEW

File: src/components/Reviews.jsx [hunk 1]
```diff
@@ -30,17 +43,57 @@ export default function Reviews() {
...
         <Tooltip id="filter-tip" title="Click here to filter by category.">
           <Link to="/api/categories">
             <IconButton>
               <FilterListIcon />
             </IconButton>
           </Link>
         </Tooltip>
+
+        <Box sx={{ minWidth: 120 }}>
+          <FormControl fullWidth>
```
Comment (HannahBurks) [inline src/components/Reviews.jsx:64]: We want to make sure that reviews are sorted by default to **date** **descending** - so we may want the drop down to default to those upon loading the page.

File: src/components/Reviews.jsx [hunk 2]
```diff
@@ -30,17 +43,57 @@ export default function Reviews() {
...
+            >
+              <MenuItem value={"votes"}>Votes</MenuItem>
+              <MenuItem value={"created_at"}>Date</MenuItem>
+              <MenuItem value={"comment_count"}>Comments</MenuItem>
+            </Select>
+          </FormControl>
+        </Box>
+
+        <FormControl>
+          <RadioGroup row defaultValue="desc">
```
Comment (HannahBurks) [inline src/components/Reviews.jsx:81]: This sorting feature is great, but we also want to make sure that it works for both all reviews and reviews by category.

**Review by HannahBurks — CHANGES_REQUESTED** [review]
Great work Alice! It looks really good so far 😄 Once the requested changes have been made, please send another PR.

**Comment by captainread** [comment]
Sort now works for Reviews and FilteredReviews, but in a very not "DRY" way. Tried to extract it into its own component and things got difficult. I want to work on the rest of my tickets first and if I have time I will come back to this! Thanks!!

**Review by yalicee — APPROVED** [review]
V happy with your code :)

**Pull request merged**